mm2src/coins/tendermint/tendermint_coin.rs

```diff
@@ -1016,7 +1025,9 @@ impl TendermintCoin {
 
             let raw_response = self.rpc_client().await?.perform(request).await?;
 
-            if raw_response.response.log.to_string().contains(ACCOUNT_SEQUENCE_ERR) {
+            let log = raw_response.response.log.to_string();
+            if log.contains(ACCOUNT_SEQUENCE_ERR) {
+                account_info.sequence = parse_expected_sequence_number(&log)?;
```

Comment:
Q: Is the entity sending this error message the same as that used to suggest the seq number?

Reply:
The question doesn't seem clear to me. Previously the node didn't suggest any number. We were trying to read it from account information RPC.

Reply:
Cleared Up over DM